go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
 	github.com/opencontainers/selinux v1.8.0
 	github.com/pkg/errors v0.9.1
-	github.com/seccomp/libseccomp-golang v0.9.1
+	github.com/seccomp/libseccomp-golang v0.9.2-0.20200616122406-847368b35ebf
```

Comment:
Not directly related to this PR, but could you include https://github.com/seccomp/libseccomp-golang/commit/3879420cc921efb4f1a2d75bea84e6158ef21a78 (`all: handle EINTR and ENOENT`)

Reply:
Done, vendored the latest master  :+1:

Reply:
thanks